docs/source/topic/allowing.md

```diff
@@ -0,0 +1,142 @@
+(allowing)=
+
+# Allowing access to your JupyterHub
+
+OAuthenticator is about deferring **authentication** to an external source,
+assuming your users all have accounts _somewhere_.
+But many of these sources (e.g. Google, GitHub) have _lots_ of users, and you don't want _all_ of them to be able to use your hub.
+This is where **authorization** comes in.
+
+In OAuthenticator, authorization is represented via configuration options that start with `allow` or `block`.
+
+There are also lots of OAuth providers, and as a result, lots of ways to tell OAuthenticator who should be allowed to access your hub.
+
+## Default behavior: nobody is allowed!
+
+The default behavior of OAuthenticator (starting with version 16) is to block all users unless explicitly authorized via _some_ `allow` configuration.
+If you want anyone to be able to use your hub, you must specify at least one `allow` configuration.
+
+```{versionchanged} 16
```

Comment:
This isn't accurate as it is written now, if for example github configured allowed_orgsnizations without configuring allowed_users, it would mean that allow_all still wasn't implied.

Rather whats accurate (i believe) is that allow_all was implied if no other config starting with allow was specified.

Reply:
Right, it should be "no other `allow` config", not just `allowed_users`. Got my JupyterHub and OAuthenticator mixed up.